README.md

```diff
@@ -1 +1,164 @@
-#B.CfnOpensearchIndex
+# B.CfnOpenSearchIndex
+
+**b-cfn-opensearch-index** - AWS CloudFormation custom resource that handles the creation, update and removals of empty OpenSearch indexes.
+
+### Description
+
+While you can create an index by using a document as a base, using this AWS CloudFormation custom resource, you can create an empty index for use later. Update of previously created index possible too, there are some limitations (please refer to "Known limits"). Reindex of documents triggers by default and prevents data loss in case of the index name change. In case of destruction of AWS CloudFormation stack containing this custom resource, removal of created indexes initiates and removes all previously created indexes with the idea no one leaves behind.<br><br> **Attention** remove of indexes destroy all documents indexed using that index.
```

Comment:
While you can create an OpenSearch index just by using a document as a base, using this AWS CDK resource lets you create an empty index for later use. 
The resource supports **update** action, however, there are some limitations (please refer to "Known limits"). Update triggers reindexing of documents by default to prevent data loss. 
The resource also supports **delete** action, where removal of created index removes all previously created indexes. **Attention** removal of indexes destroys all indexed documents.